src/intTest/resources/edifact/pathology_2.edifact.dat

```diff
@@ -38,7 +38,7 @@ FTX+CID+++TIRED ALL THE TIME, LOW Hb'
 FTX+CID+++COELIAC'
 S16+16'
 SEQ++1'
-SPC+TSP+:::VENOUS BLOOD'
+SPC+TSP+T016:920'
```

Comment:
Can we have this example having both text and code?

Reply:
Sure. Is that to be more realistic, or to better show off (test) the range of functionality?

Reply:
To show both are mapped